Web/starter_project_vanguard/tailwind.config.js

```diff
@@ -11,21 +11,22 @@ module.exports = {
       colors: {
         primary: '#264FAD',
         accent: '#FF9F43',
+        aboutbackground: '#FDFEFF',
```

Comment:
[nit] the background color for most the pages is the same so can you give it a more inclusive name?